.github/workflows/scrape.yml

```diff
@@ -0,0 +1,47 @@
+name: ⚙️ Scrape
+
+on:
+  workflow_call:
+    inputs:
+      object_type:
+        description: What objects to scrape (people, events, bills)?
+        type: string
+      object_ids:
+        description: Specific entities to scrape (comma-separated)?
+        type: string
+      window:
+        description: How many days to scrape?
+        type: string
+      rpm:
+        description: How many requests per minute?
+        type: string
```

Comment:
[You can also add defaults if you'd like.](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs)